### DISCUSSION TITLE
No access on Win10 PC --need help

### DISCUSSION THREAD

**Discussion by GreyMoth** [45Drives/cockpit-file-sharing#66] [Q&A] [answered]
Hi all,

I have installed this onto my cockpit instance and followed all instructions-- from the server side, graphically, everything looks good. On the windows computer I intend to access this samba server from, it does not show up in the "Network" assets. Attempting to create a new network resource provides error 'The folder you entered does not appear to be valid...' I have confirmed that I can ping the server from the w10 pc and vice versa. As a workaround attempt, I enabled smb 1.0 feature on the windows computer, but still no luck. I don't believe there is any firewall configured on the server (ufw not installed). I'm not sure what else I need to do to resolve this issue. Help me please.

Server info:
uname -a
Linux myserver 5.15.0-53-generic #59-Ubuntu SMP Mon Oct 17 18:53:30 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by bkelly16** [comment, marked as answer]
can you provide your samba configuration.
`net conf list`

Have you tried mapping the drive or navigating to the network path in explorer ?
![windows_network_path](https://user-images.githubusercontent.com/19629309/204095879-4375d935-d043-4120-9aba-80fd7d3df737.png)
Samba will not show up in "Network" tab of windows by default, you need a helper service `wsd`.

See [this](https://devanswers.co/discover-ubuntu-machines-samba-shares-windows-10-network/) blog for details on why it is off by default and what you need to do to fix it.

EDIT

Blog references Ubuntu. See [wsdd](https://github.com/christgau/wsdd#installation) github for installing on other distros.

**Comment by GreyMoth** [comment]
Miraculous. Thank you. 
The blog post fixed my problem!

I spoke too soon :/ 
The server shows in network path now, but still cannot be accessed. 
'Network Error
Windows cannot access \\myserver'

Unfortunately, using the \\ip\share does not allow access. 

My samba configuration:

`[global]
        log file = /var/log/samba/log.%m
        max log size = 1000
        logging = file
        panic action = /usr/share/samba/panic-action %d
        server role = standalone server
        obey pam restrictions = yes
        unix password sync = yes
        passwd program = /usr/bin/passwd %u
        passwd chat = *Enter\snew\s*\spassword:* %n\n *Retype\snew\s*\spassword:* %n\n *password\supdated\ssuccessfully* .
        pam password change = yes
        map to guest = bad user
        usershare allow guests = yes
        server string = myserver
        workgroup = WORKGROUP

[fileshare]
        comment = fileshare
        path = /srv/samba/
        browseable = yes
        guest ok = no
        read only = no
        create mask = 0755`


Thanks for your help Bkelly16

**Reply by GreyMoth** [reply]
Update:
https://technoresult.com/fix-cant-access-this-shared-folder-because-your-organizations-security-policies/

This was my fix to access the share.
